download/download.go

```diff
@@ -276,6 +271,8 @@ func (d *Downloader) download(ctx context.Context, m metrics.Metrics) (*download
 			t := int64(0)
 			d.client = d.client.SetResponseHeaderTimeout(&t)
 		}
+	} else {
+		d.client = d.client.WithHeader("Prefer", fmt.Sprintf("wait=0"))
```

Comment:
The linter output is hard to read sometimes. It wants this to be
```suggestion
		d.client = d.client.WithHeader("Prefer", "wait=0")
```

Reply:
Oh yes sorry :(.